project-generator/src/test/java/io/syndesis/project/converter/DefaultProjectGeneratorTest.java

```diff
@@ -73,7 +74,10 @@
 @RunWith(Parameterized.class)
 public class DefaultProjectGeneratorTest {
     private static final String CONNECTORS_VERSION = ResourceBundle.getBundle("test").getString("connectors.version");
-    private static final ConnectorCatalogProperties CATALOG_PROPERTIES = new ConnectorCatalogProperties();
+    private static MavenProperties mavenProperties = new MavenProperties(map("maven.central", "https://repo1.maven.org/maven2",
+        "redhat.ga", "https://maven.repository.redhat.com/ga",
```

Comment:
What part is using the redhat repo ?

Reply:
Just the test to make sure that if we need a repository it would get added to the integration POM, that part was refactored from static initializer.